src/tests/admin/resources/AssociatedRoomIsDisplayedInResourceAssociationPage.java

```diff
@@ -0,0 +1,107 @@
+package tests.admin.resources;
+
+import static framework.common.AppConfigConstants.EXCEL_INPUT_DATA;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+import org.json.JSONException;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import framework.pages.admin.HomeAdminPage;
+import framework.pages.admin.conferencerooms.RoomInfoPage;
+import framework.pages.admin.conferencerooms.RoomResourceAssociationsPage;
+import framework.pages.admin.conferencerooms.RoomsPage;
+import framework.pages.admin.resources.ResourceAssociationsPage;
+import framework.pages.admin.resources.ResourceCreatePage;
+import framework.pages.admin.resources.ResourceInfoPage;
+import framework.pages.admin.resources.ResourcesPage;
+import framework.rest.RootRestMethods;
+import framework.utils.readers.ExcelReader;
+
+/**
+ * TC33: Verify that rooms name associated to a resource are displayed on {Name} column in 
+ * {ResourceInfo>Resource Associations} form when it is selected
+ * @author Marco Llano
+ */
+public class AssociatedRoomIsDisplayedInResourceAssociationPage {
+	private ResourceAssociationsPage resourceAssociationsPage;
+	private ResourcesPage resourcesPage;
+	private HomeAdminPage homeAdminPage = new HomeAdminPage();
```

Comment:
not initialize the variable "homeAdminPage" here (only private HomeAdminPage homeAdminPage;)